api/app/Console/Commands/ExpireOldDigitalCareersPoolCandidates.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Pool;
+use Database\Helpers\ApiEnums;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+
+class ExpireOldDigitalCareersPoolCandidates extends Command
+{
+    // this command updates all candidates associated with pool of key = digital_careers as expired
+    // this does so by expiry date and pool candidate status, a double approach as status determination is changing
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'expire:old_digital_pool_candidates';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Expire the candidates of the old digital careers pool.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        // fill variables then execute the database statement with them
+        $expiredStatus = ApiEnums::CANDIDATE_STATUS_EXPIRED;
+        $oldDigitalCareersId = Pool::where('key', 'ilike', 'digital_careers')->first()->toArray()['id'];
+
+        DB::update("update pool_candidates
```

Comment:
Don't forget to update the `updated_at` timestamp, too.  (UTC, remember)

Reply:
got stuff
french translation https://github.com/GCTC-NTGC/gc-digital-talent/pull/4467/commits/8f8f229b615ef684a697e7e7dfab66d49b546521
updated_at here https://github.com/GCTC-NTGC/gc-digital-talent/pull/4467/commits/467b8ca27f65f3bb0abd7cc21074275890377b73
pool owner here https://github.com/GCTC-NTGC/gc-digital-talent/pull/4467/commits/66da2ae68eda458557e1221de45abb023e0cee5b